.gitignore

```diff
@@ -1,4 +1,5 @@
 target/
+.idea/
```

Comment:
Please include .idea to your [global ignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) as otherwise we end up with ignores for every possible editor out there :)